es-vue-base/src/lib-components/EsNavBarTopLevelMenu.vue

```diff
@@ -5,43 +5,33 @@
         <div class="top-header-inner w-100">
             <!-- desktop fly-out menu trigger -->
             <es-nav-bar-link
-                v-if="!link"
-                class="nav-link dropdown-toggle d-none d-lg-block py-100 font-weight-bolder"
-                href="#"
+                class="nav-link d-none d-lg-block py-100 font-weight-bolder"
+                :class="{ 'dropdown-toggle': !link }"
+                :href="link || '#'"
                 data-toggle="dropdown"
                 aria-haspopup="true"
-                aria-expanded="false">
+                :aria-expanded="link ? 'true' : 'false'">
```

Comment:
If it's a link, I don't think it should have an `aria-expanded` attribute at all, so maybe `undefined` is a better value than `true` here.